### ISSUE TITLE
Mailing List / Groups

### ISSUE THREAD

**Issue by aspdye** [owncloud/mail#772]
Can you create an enhancement, which allows you to write emails to OwnCloud-Contact Groups?
That would be nice for creating mailing lists.

**Comment by DeepDiver1975** [comment]
Thanks a lot - putting this to the backlog for the time being - contributions to fix this is are any time welcome. THX

**Comment by jancborchardt** [comment]
Yeah, good call. Contact groups should also be in the address autocompletion (much like for the share dialog). A first implementation could simply insert all the individual mail addresses in the case of selecting a group from the autocompletion.

**Comment by aspdye** [comment]
@jancborchardt I think this feature is verry essential. Much more as an calendar integration (which has the milestone 0.4)

**Comment by jancborchardt** [comment]
@aspdye everyone’s priority assumption is different. It always depends on if there is someone actually doing the work. ;) If you can do it, or find someone to do it that would be great.

**Comment by virgilekat** [comment]
I'm helping in a association, and that feature could be really nice to have. If it's possible I can help to develop it ?

**Comment by ChristophWurst** [comment]
@virgilekat that would be awesome! You're welcome to join us on IRC at #owncloud-mail on freenode. Let me know if you need help. Thanks.

**Comment by virgilekat** [comment]
@ChristophWurst Ok thks ! I'm going to have a look at it sunday.

**Comment by Toub** [comment]
This feature would be a great addition to owncloud, is there any work in progress?

**Comment by virgilekat** [comment]
Unfortunately I could have time to take care of it but, it looks like someone had   [](https://github.com/owncloud/mail/issues/1531)
